src/Symfony/Component/Form/Extension/Core/DataTransformer/UlidToStringTransformer.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\Uid\Ulid;
+
+/**
+ * Transforms between a ULID string and a Ulid object.
+ *
+ * @author Pavel Dyakonov <wapinet@mail.ru>
+ */
+class UlidToStringTransformer implements DataTransformerInterface
+{
+    /**
+     * Transforms a Ulid object into a string.
+     *
+     * @param Ulid $ulid A Ulid object
+     *
+     * @return string A value as produced by Uid component
+     *
+     * @throws TransformationFailedException If the given value is not an Ulid object
+     */
+    public function transform($ulid)
```

Comment:
```suggestion
    public function transform($value)
```
I would prefer value here

Reply:
fixed